app/guid-file/template.hbs

```diff
@@ -67,21 +67,21 @@
                 </div>
             {{/if}}
             {{#if this.metadataOpened}}
-                <div local-class='right-container {{if this.isMobile 'Mobile'}}'>
+                <div local-class='right-container {{if this.isMobile 'Mobile'}}' data-test-metadata-tab>
                     <FileMetadataManager @file={{this.model.fileModel}} as |manager|>
                         <div local-class='metadata-title-edit-download'>
-                            <h2 local-class='metadata-heading'>
+                            <h2 local-class='metadata-heading' data-test-metadata-header>
                                 {{t 'file_detail.file_metadata'}}
                             </h2>
                             <div local-class='edit-download-buttons'>
                                 {{#unless manager.inEditMode}}
                                     {{#if manager.userCanEdit}}
                                         <Button
-                                            data-test-edit-metadata-button
-                                            aria-label={{t 'general.edit'}}
-                                            data-analytics-name='Edit metadata'
-                                            @layout='fake-link'
-                                            {{on 'click' manager.edit}}
+                                          data-test-edit-metadata-button
+                                          aria-label={{t 'general.edit'}}
+                                          data-analytics-name='Edit metadata'
+                                          @layout='fake-link'
+                                          {{on 'click' @manager.edit}}
```

Comment:
the `@` isn't needed on this one, since `manager` in this case refers to the FileMetadataManager variable defined in line 71, not a passed in argument
```suggestion
                                          {{on 'click' manager.edit}}
```